### DISCUSSION TITLE
Change redirect after login

### DISCUSSION THREAD

**Discussion by hellodit** [orchidsoftware/platform#1554] [Q&A] [answered]
Hii, I'm new here! 

I got confused about how to change the redirect page after login, in my case I want to change the redirect to `admin` instead of the default page `home`. 

Thanks for helping me, appreciate it

**Comment by tabuna** [comment, marked as answer]
You can change this in your laravel delivery:

https://github.com/laravel/laravel/blob/6d082c81e5bf226dfa6d1dc030e69a225349f535/app/Providers/RouteServiceProvider.php#L11-L20

**Reply by hellodit** [reply]
Thanks!

**Reply by noum** [reply]
Doesn't seem to work anymore, the variable is just ignored

**Reply by tabuna** [reply]
@nomentsoa-andrianjatovo You can specify the main screen name that you want users to go to in the configuration file:

https://github.com/orchidsoftware/platform/blob/058eef0cf755d842e82f5edbbf5847f6b981e563/config/platform.php#L88-L98

**Reply by noum** [reply]
Is there a way to redirect once to a custom page only after the login action but keep the links of the logo and other links intact? Bc I thought that was the aim of the variable in RouteServiceProvider.php initially, then it seemed to have stopped working afterward if I'm not mistaken.
